pkg/vss/vehicle-structs.go

```diff
@@ -86,6 +86,8 @@ const (
 	FieldOBDRunTime = "obdRunTime"
 	// FieldOBDShortTermFuelTrim1 PID 06 - Short Term (immediate) Fuel Trim - Bank 1 - negative percent leaner, positive percent richer
 	FieldOBDShortTermFuelTrim1 = "obdShortTermFuelTrim1"
+	// FieldOBDStatusDTCCount Number of Diagnostic Trouble Codes (DTC)
+	FieldOBDStatusDTCCount = "obdStatusDTCCount"
```

Comment:
lgtm, I was confused first why we need to have FieldOBDStatusDTCCount but not just FieldOBDDTCCount, but it seems that in VSS it is defined under the Status structure.